lua/harpoon/init.lua

```diff
@@ -43,15 +43,15 @@ function merge_table_impl(t1, t2)
     end
 end
 
-function merge_tables(...) 
```

Comment:
Is my remove whitespace not being called?  Double you Tee Eff!!! let me fix that.

Reply:
Seems like it. I did remove in several files.